rrdtool/rrd_mts_similarity.c

```diff
@@ -58,22 +59,22 @@ u_int verbose = 0, similarity_threshold = 100, skip_zero = 0;
 /* *************************************************** */
 
 static void help() {
-  printf("Usage: rrd_similarity [-v][-a <alpha>][-e <end>][-q][-s <start>]\n"
-	 "                      -f <filename> -d <basedir> [-t <threshold>]\n"
+  printf("Usage: rrd_mts_similarity [-v][-a <alpha>][-e <end>][-q][-s <start>]\n"
+	 "                      -f <filename_1> <filename_2> ... <filename_n> -d <basedir> [-t <threshold>]\n"
 	 "-a             | Set alpha. Valid range >0 .. <1. Default %.2f\n"
 	 "-e <end>       | RRD end time. Default %s\n"
 	 "-q             | Quick output (only anomalies are reported)\n"
 	 "-s <start>     | RRD start time. Default %s\n"
 
 	 "-d <basedir>   | Base directory where RRD filename is searched\n"
-	 "-f <rrd path>  | Path of the RRD filename to analyze\n"
+	 "-f <rrd_path1> <rrd_path2> ...  | Path of the RRDs filename to analyze\n"
```

Comment:
new explanation of how -f works

Reply:
Formatting is broken and <rrd_path1> <rrd_path2> should be [<rrd_path>]+

Reply:
Reworked -f in commit 108daa5